src/modules/elevationProfile/components/ElevationProfile.js

```diff
@@ -423,16 +424,16 @@ export class ElevationProfile extends MvuElement {
 	/**
 	 * @private
 	 */
-	async _getAltitudeProfile(coordinates) {
+	async _getElevationProfile(coordinates) {
+		const translate = (key) => this._translationService.translate(key);
 		if (Array.isArray(coordinates) && coordinates.length >= 2) {
 			try {
 				const profile = await this._elevationService.getProfile(coordinates);
 				this._enrichProfileData(profile);
 				this.signal(Update_Profile_Data, profile);
 			} catch (e) {
-				console.warn(e.message);
-				// Todo: emit error notification
-				// this.signal(Update_Profile_Data, null);
+				console.error(e);
+				emitNotification(translate('elevationProfile_could_not_load'), LevelTypes.ERROR);
```

Comment:
The ElevationProfile won't be (fully) opened when an exception occurs (in order to display an "empty" profile). Therefore we have to call also `this.signal(Update_Profile_Data, EmptyProfileData);` for that case.